Translation/README.md

```diff
@@ -60,14 +67,12 @@ flowchart LR
 
 This Translation use case performs Language Translation Inference across multiple platforms. Currently, we provide the example for [Intel Gaudi2](https://www.intel.com/content/www/us/en/products/details/processors/ai-accelerators/gaudi-overview.html) and [Intel Xeon Scalable Processors](https://www.intel.com/content/www/us/en/products/details/processors/xeon.html), and we invite contributions from other hardware vendors to expand OPEA ecosystem.
 
-## Deploy Translation Service
-
-The Translation service can be effortlessly deployed on either Intel Gaudi2 or Intel Xeon Scalable Processors.
-
-### Deploy Translation on Gaudi
-
-Refer to the [Gaudi Guide](./docker_compose/intel/hpu/gaudi/README.md) for instructions on deploying Translation on Gaudi.
+## Deployment Options
 
-### Deploy Translation on Xeon
+The table below shows different deployment options to choose from. They outline in detail the implementation of this example on the selected hardware.
```

Comment:
The table below lists the available deployment options and their implementation details for different hardware platforms.